content/rest/reference/activity.md

```diff
@@ -56,7 +56,7 @@ To get a feed in Atom format, you must specify the `application/atom+xml` type i
 #### Response
 
 ```shell
-Status: 200 OK
+HTTP/1.1 200 OK
```

Comment:
@skedwards88 I don't think that's correct, since this shows output from an actual `curl` response and this header doesn't exist. So given that, I think the `HTTP/1.1 200 OK` is better since that's at least in the response for an HTTP/1.1 response.

Otherwise we can also remove the whole line in it's entirely but I don't want to leave this one in there to be potentially confusing.

Reply:
Thanks for explaining! Changing to the actual curl response makes sense.